### ISSUE TITLE
Tinc1.1 Binaries

### ISSUE THREAD

**Issue by lancethepants** [Vilbrekin/tinc_gui#25]
Would be cool to have the option to pick either tinc1.0 or tinc1.1 prerelease X.

tinc1.1 for purposes of this project would only need the 'tincd' binary. The 'tinc' binary would be unnecessary.
You wouldn't need the additional library dependencies ncurses and readline either, just the same as tinc 1.0; openssl, zlib, lzo.

I realize tinc1.1preX and tinc1.0 should play nice together. tinc1.1pre10 release seems to have had a lot of issues though and won't play nice with tinc_gui tinc1.0 version ( at least for me ).

Pretty minimal I think on gui changes, and then just have to pre-compile another binary to package. 
If your not to keen on doing it yourself, I'd be willing to give it a try if such a pull request would be favorably looked upon. (I'd like to think I could figure it out anyway).

**Comment by calisro** [comment]
Just my 2 cents.  I run a 1.1preX (currently 11 which works great btw) server and i'm running this 1.0 version on my android devices.  It would be nice to be able to install the pre releases for android to match up to my server (separate branch or selectable in GUI)  or at least be able to manually download a pre-compiled tinc executable for use at my own risk of the gui not working.  If not, is there a guide/doc on how to compile this on our own?  I can try to figure it out, but if you have something, that would be nice.  Thanks for this fork..

**Comment by luckyhacky** [comment]
You have to download Android NDK (not the SDK!) because you have to compile C code and not only the Java code.

Second step: see readme of Tincat https://github.com/gsliepen/tinc -> https://github.com/gsliepen/tinc/blob/master/README.android

After compiling you can simple download the new binary to your android phone.

Other option is to checkout 1.1 branch of  lancethepants at https://github.com/lancethepants/tinc_gui
and compile it.

**Comment by Vilbrekin** [comment]
I've started merging hwhw branch into tinc-1.1. Tincd is starting, however stop functionality is not working yet, as pid file's content and rights changed with 1.1.

**Comment by calisro** [comment]
Yes @luckyhacky , I cloned gsliepen's 1.1 branch and compiled tinc/tincd.  Had to ifdef some code to get it to compile but it works great.
